template-marketplace/hertzbeat-template-hub/src/main/java/org/apache/hertzbeat/templatehub/sureness/config/SurenessConfiguration.java

```diff
@@ -0,0 +1,122 @@
+package org.apache.hertzbeat.templatehub.sureness.config;
+
+import com.usthe.sureness.matcher.DefaultPathRoleMatcher;
+import com.usthe.sureness.matcher.PathTreeProvider;
+import com.usthe.sureness.matcher.TreePathRoleMatcher;
+import com.usthe.sureness.mgt.SurenessSecurityManager;
+import com.usthe.sureness.processor.DefaultProcessorManager;
+import com.usthe.sureness.processor.Processor;
+import com.usthe.sureness.processor.ProcessorManager;
+import com.usthe.sureness.processor.support.JwtProcessor;
+import com.usthe.sureness.processor.support.NoneProcessor;
+import com.usthe.sureness.processor.support.PasswordProcessor;
+import com.usthe.sureness.provider.SurenessAccountProvider;
+import com.usthe.sureness.provider.annotation.AnnotationPathTreeProvider;
+import com.usthe.sureness.provider.ducument.DocumentPathTreeProvider;
+import com.usthe.sureness.subject.SubjectFactory;
+import com.usthe.sureness.subject.SurenessSubjectFactory;
+import com.usthe.sureness.subject.creater.BasicSubjectServletCreator;
+import com.usthe.sureness.subject.creater.JwtSubjectServletCreator;
+import com.usthe.sureness.subject.creater.NoneSubjectServletCreator;
+import com.usthe.sureness.util.JsonWebTokenUtil;
+import org.apache.hertzbeat.templatehub.sureness.processor.CustomTokenProcessor;
+import org.apache.hertzbeat.templatehub.sureness.subject.CustomPasswdSubjectCreator;
+import org.apache.hertzbeat.templatehub.sureness.subject.CustomTokenSubjectCreator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * sureness config
+ * @author tomsun28
+ * @date 22:40 2020-03-02
+ */
+@Configuration
+public class SurenessConfiguration {
```

Comment:
Hi, suggest use the sureness springboot stater `spring-boot3-starter-sureness` and no need to config the custom SurenessConfiguration.
After import stater, need config in application.yml. Then you can use the jwt which system issue to access backend api. 
```
sureness:
  container: jakarta_servlet
  auths:
    - jwt
  jwt:
    secret: 'CyaFv0bwq2Eik0jdrKUtsA6bx3sDJeFV643R
             LnfKefTjsIfJLBa2YkhEqEGtcHDTNe4CU6+9
             8tVt4bisXQ13r4N0oxhUZRd3M6EByXIO+SV5
             dKhaX0cs3OC5lCxq20yhmUea6H6JIpSE2Rwp'
```

Btw, suggest we fix this pr review and action failed first to meet `ospp` merge requirements. For login feature we can reopen a new branch pr later to code it.

Reply:
Hi, `sureness` has now been properly inherited